test/powershell/Modules/Microsoft.WSMan.Management/CredSSP-nonAdmin.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+Describe "CredSSP cmdlet tests non-admin" -Tags 'Feature' {
+
+    $isAdmin = (New-Object Security.Principal.WindowsPrincipal ([Security.Principal.WindowsIdentity]::GetCurrent())).IsInRole([Security.Principal.WindowsBuiltinRole]::Administrator)

```

Comment:
1. Does this line work on non-Windows platforms?
2. Code should not be [outside of a Pester block](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#free-code-in-a-describe-block). This should be moved within a BeforeAll
3. Check out the guidelines on [Admin for Windows](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#admin-privileges-in-tests)
4. Consider using the [pattern for skipping tests](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md#skipping-tests-in-bulk) as well.